docs/src/routes/docs/components/tab.svx

```diff
@@ -0,0 +1,98 @@
+---
+name: Tab
+relatedComponents:
+  - name: Tabs
+    link: ./docs/components/tabs
+---
+
+<script>
+  import { Tab } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+</script>
+
+# Tab
+
+Allows for finer control over individual tabs when the default [==`Tabs`==](./docs/components/tabs) is not good enough.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Tab value="page1" name="nav1">
+      Components
+    </Tab>
+    <Tab value="page2" name="nav1" disabled>
+      Installation
+    </Tab>
+    <Tab value="Showcase" name="nav1" />
+  </div>
+  <div slot="source">
+
+```svelte
+<Tab value="page1" name="nav1">
+  Components
+</Tab>
+<Tab value="page2" name="nav1" disabled>
+  Installation
+</Tab>
+<Tab value="Showcase" name="nav1" />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`ripple`** | `true` | `boolean` | Adds a ripple to the tab when clicking on it. |
```

Comment:
Perhaps we should invert this prop for consistency with the rest of the library, to `noRipple` instead